src/Conversion/ONNXToKrnl/Math/Elementwise.cpp

```diff
@@ -322,6 +316,55 @@ struct ScalarOp<ONNXTanOp> {
   using IOp = NotSuportedScalarOp;
 };
 
+//===----------------------------------------------------------------------===//
+// Scalar unary ops for lowering ONNXIsInfOp
+//===----------------------------------------------------------------------===//
+template <>
+struct ScalarOp<ONNXIsInfOp> {
+  using FOp = CustomScalarOp;
+  using IOp = NotSuportedScalarOp;
+};
+
+template <>
+Value emitScalarOpFor<ONNXIsInfOp>(ConversionPatternRewriter &rewriter,
```

Comment:
It seems computation here can be simdized. Could you define `simdAnalysis` for this op for SIMD?

Reply:
I have never done computation for SIMD but I can try and you all can let me know if what I have makes sense. Thanks

Reply:
Okay added `simdAnalysis`...let me know if this looks good. Thanks

Reply:
it seems that the code is "equal" and "or"... so why `genericops::IsInfGop`

Reply:
> it seems that the code is "equal" and "or"... so why `genericops::IsInfGop`

Learning... so from looking at the code, I would use`LogicalGop`... /* All logical ops: and, or, xor, not, nor, nand,... */